test/e2e/integration/001-hello-gutenberg.js

```diff
@@ -1,13 +1,9 @@
 describe( 'Hello Gutenberg', () => {
 	before( () => {
-		cy.login();
+		cy.newPost();
```

Comment:
I explicitly decided to avoid the `newPost` command in this test. Usually, when you have commands like those. You have one test performing this manually and an optimized version as a command to reuse across other tests. The reusable version can use any trick (fake APIs, direct URL access to speed up the behavior).

For this one specifically, I think it's fine to drop the manual testing, it's not that important but I just wanted to explain.

Reply:
>You have one test performing this manually and an optimized version as a command to reuse across other tests.

Yeah, that made sense, with one of the ideas of this pull request being that if we can consolidate the optimization within `newPost`, `visitAdmin` (i.e. try navigating directly, handle login) then the individual tests benefit by having a consistent interface and not relying on any other tests to have handled specific steps.